Gemfile.jruby-2.6.lock.release

```diff
@@ -568,6 +568,7 @@ GEM
       logstash-codec-json
       logstash-codec-plain
       logstash-core-plugin-api (>= 2.1.12, <= 2.99)
+      rexml
```

Comment:
Ah, new dependency... where it is coming from?! I would be careful adding `.xml` dependencies since we are struggling to overcome some snakeyaml CVEs :(